doc/assy.rst

```diff
@@ -205,7 +205,8 @@ Final result
 
 Below is the complete code including the final solve step.
 
-.. code-block:: python
+.. cq_vtk::
```

Comment:
Is there any reason to use cq_vtk over the plain cadquery directive used everywhere else?

Reply:
Good catch, fixed.